build.nix

```diff
@@ -1,15 +1,18 @@
-{ lib, rustPlatform, rust, buildInputs, nativeBuildInputs, ... }:
+{ lib, rustPlatform, florestaRust, buildInputs }:
 
 let
-  pname = "florestad";
-  version = "0.5.1";
-
-  buildRustPackage = rustPlatform.buildRustPackage.override {
-    rustc = rust;
-    cargo = rust;
-  };
+    # Pname defines the name of the package and will decide the output of this expression
+    pname = "floresta-node";
+    version = "0.6.0";
```

Comment:
Thinking out loud: We need a script to update all versions on release, I never remember to bump all places 😆